styles/styles.css

```diff
@@ -77,26 +49,66 @@ header {
   height: var(--nav-height);
 }
 
+h1, h2, h3, h4, h5, h6,
+.h1, .h2, .h3, .h4, .h5, .h6 {
+  margin-bottom: 20px;
+  margin-top: 0;
+}
+
 h1,
+.h1 {
+  font-size: 1.8063rem;
+  line-height: 1.2;
+  font-weight: 400;
+  text-transform: uppercase;
+}
+
 h2,
+.h2 {
+  font-size: 1.7rem;
+  line-height: 1.3;
+  font-weight: 400;
+}
+
 h3,
+.h3 {
+  font-size: 1.5rem;
+  line-height: 1.4;
+  font-weight: 400;
+}
+
 h4,
+.h4 {
+  font-size: 1.25rem;
+  line-height: 1.4;
+  font-weight: 600;
+}
+
 h5,
-h6 {
-  font-family: var(--heading-font-family);
+.h5 {
+  font-size: 16px;
```

Comment:
Can you make this rem as rest of them are in rem?

Reply:
Done